src/main/java/de/berlin/lostberlin/service/MailService.java

```diff
@@ -0,0 +1,56 @@
+package de.berlin.lostberlin.service;
+
+import com.sendgrid.*;
+import de.berlin.lostberlin.model.Business;
+import de.berlin.lostberlin.model.Order;
+import de.berlin.lostberlin.repository.BusinessRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.Optional;
+
+public class MailService {
+    @Autowired
+    private static BusinessRepository businessRepo;
+
+    public static Response sendEmail(Order order, MailTypes mailType) throws IOException, SQLException {
+        Email from = new Email("aoneko@gmx.com");
```

Comment:
It is better not to push IOException out since the caller has no idea what to do with it. For the caller it is probably enough to get one exception that something went wrong and do not distinguish much about them.

Reply:
If I see it right, it might be important for the client to distinguish between these two exceptions:
- SQL Exception:  order won't be sent to the businesses, so that the client needs to repeat the procedure;
- IO Exception: order is sent to the businesses, but the confirmation e-mail can't be sent.